src/utilities/createRouter.ts

```diff
@@ -7,7 +7,7 @@ import { createRouterPush } from '@/utilities/createRouterPush'
 import { createRouterReject } from '@/utilities/createRouterReject'
 import { createRouterResolve } from '@/utilities/createRouterResolve'
 
-export function createRouter<T extends Routes>(routes: T, options: RouterOptions = {}): Router<T> {
+export function createRouter<const T extends Routes>(routes: T, options: RouterOptions = {}): Router<T> {
```

Comment:
This didn't really fix anything but makes sense in case users inline their routes.